tests/foreman/api/test_hostcollection.py

```diff
@@ -15,446 +15,457 @@
+    :BZ: 1325989
+    """
+    hosts = fake_hosts
+    host_collection = entities.HostCollection(host=hosts, organization=module_org).create()
+    assert len(host_collection.host) == len(hosts)
 
-        :CaseImportance: Critical
 
-        :BZ:1325989
-        """
-        host_collection = entities.HostCollection(host=self.hosts, organization=self.org).create()
-        self.assertEqual(
-            frozenset(host.id for host in host_collection.host),
-            frozenset(host.id for host in self.hosts),
-        )
+@pytest.mark.tier2
+def test_positive_add_host(module_org, fake_hosts):
+    """Add a host to host collection.
 
-    @pytest.mark.tier1
-    def test_positive_update_name(self):
-        """Check if host collection name can be updated
+    :id: da8bc901-7ac8-4029-bb62-af21aa4d3a88
 
-        :id: b2dedb99-6dd7-41be-8aaa-74065c820ac6
+    :expectedresults: Host was added to the host collection.
 
-        :expectedresults: Host collection name was successfully updated
+    :CaseLevel: Integration
 
-        :CaseImportance: Critical
-        """
-        host_collection = entities.HostCollection(organization=self.org).create()
-        for new_name in valid_data_list():
-            with self.subTest(new_name):
-                host_collection.name = new_name
-                self.assertEqual(host_collection.update().name, new_name)
+    :BZ:1325989
+    """
+    hosts = fake_hosts
+    host_collection = entities.HostCollection(organization=module_org).create()
+    host_collection.host_ids = [hosts[0].id]
+    host_collection = host_collection.update(['host_ids'])
+    assert len(host_collection.host) == 1
 
-    @pytest.mark.tier1
-    def test_positive_update_description(self):
-        """Check if host collection description can be updated
 
-        :id: f8e9bd1c-1525-4b5f-a07c-eb6b6e7aa628
+@pytest.mark.upgrade
+@pytest.mark.tier2
+def test_positive_add_hosts(module_org, fake_hosts):
+    """Add hosts to host collection.
 
-        :expectedresults: Host collection description was updated
+    :id: f76b4db1-ccd5-47ab-be15-8c7d91d03b22
 
-        :CaseImportance: Critical
-        """
-        host_collection = entities.HostCollection(organization=self.org).create()
-        for new_desc in valid_data_list():
-            with self.subTest(new_desc):
-                host_collection.description = new_desc
-                self.assertEqual(host_collection.update().description, new_desc)
+    :expectedresults: Hosts were added to the host collection.
 
-    @pytest.mark.tier1
-    def test_positive_update_limit(self):
-        """Check if host collection limit can be updated
+    :CaseLevel: Integration
 
-        :id: 4eda7796-cd81-453b-9b72-4ef84b2c1d8c
+    :BZ: 1325989
+    """
+    hosts = fake_hosts
+    host_ids = []
+    host_collection = entities.HostCollection(organization=module_org).create()
+    host_ids = f"""[{','.join([f"{host.id}" for host in hosts])}]"""
+    host_ids = host_ids.strip('][').split(',')
```

Comment:
This can likely be done better. what does the final output of line 226 look like?

Reply:
@JacobCallahan 
>>> host_ids                                                                                                                                         
['125', '126']

Reply:
Ping @JacobCallahan, please pass this or make further suggestions. Thank you.

Reply:
changed to:
`    host_ids = {str(host.id) for host in fake_hosts}
`